Quake/Quake/Common/LocationServices/LocationServices.swift

```diff
@@ -0,0 +1,47 @@
+
+import Foundation
+import CoreLocation
+import UIKit
+
+class LocationServices {
+    private let locationManager = CLLocationManager()
+    
+    func checkLocationServices(controller: UIViewController) {
+        guard CLLocationManager.locationServicesEnabled() else {
+            let alert = UIAlertController(title: "Location Services not enabled", message: "Turning on location services allows us to pinpoint you in the world map so you can see earthquakes around you.", preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Close", style: UIAlertAction.Style.default, handler: nil))
+            controller.present(alert, animated: true, completion: nil)
+            return
+        }
+        
+        locationManager.delegate = controller as? any CLLocationManagerDelegate
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        
+        let alert = UIAlertController(title: "Location Services enabled", message: "Location Services enabled for Quake.", preferredStyle: UIAlertController.Style.alert)
+        alert.addAction(UIAlertAction(title: "Accept", style: UIAlertAction.Style.default, handler: nil))
+        controller.present(alert, animated: true, completion: nil)
```

Comment:
There is repeated code with alert presenting. You could create a function to showAlert with different messages and just call it from here.